PackageViewModel/PublishPackage/PublishPackageViewModel.cs

```diff
@@ -115,6 +119,27 @@ public ObservableCollection<string> PublishSources
                 }
             }
         }
+        public bool? UseApiKey
+        {
+            get { return _useApiKey; }
+            set
+            {
+                if (_useApiKey != value)
+                {
+                    _useApiKey = value;
+                    OnPropertyChanged("UseApiKey");
```

Comment:
@onovotny we can already use nameof isn't?

Reply:
`nameof` can be used, it's a compiler feature.

Reply:
I'm not familiar with the `nameof` keyword. I'll look into it and implement it

Reply:
see https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/nameof

great feature!